tsconfig.sdk.json

```diff
@@ -5,20 +5,18 @@
     "emitDeclarationOnly": true,
     "outDir": "./resources/embedding-sdk/dist",
     "paths": {
-      "*": [
-        "./frontend/src/*",
-        "./frontend/test/*",
-        "./enterprise/frontend/src/*",
-        "./enterprise/frontend/test/*"
-      ],
-      "cljs/*": ["./target/cljs_dev/*"]
+      "metabase/*": ["./frontend/src/metabase/*"],
```

Comment:
I've tried to make this config a bit cleaner, I tested that it passes the build, but not sure how to make sure it really works. I just checked the file size and it's about the same size as the one built with this config.

One thing to note is that even with the same config, building the SDK multiple times would result in different file sizes, so I hope that the build would work with my proposed config.

```json
{
  "extends": "./tsconfig.base.json",
  "compilerOptions": {
    "declaration": true,
    "emitDeclarationOnly": true,
    "outDir": "./resources/embedding-sdk/dist",
    "paths": {
      "*": ["./frontend/src/*", "./enterprise/frontend/src/*"], // Uses the same config as the the main `tsconfig.json` but removing tests directories
      "cljs/*": ["./target/cljs_release/*"]
    }
  },
  "include": [
    "frontend/src/**/*.ts", // Specify only ts files here, so we don't need to exclude JS files below.
    "frontend/src/**/*.tsx",
    "enterprise/frontend/src/**/*.ts",
    "enterprise/frontend/src/**/*.tsx"
  ],
  "exclude": [
    "node_modules",
    "**/*.unit.spec.*",
    "**/mocks/*",
    "**/tests/*",
    "**/test/*",
    "**/*test*", // catch-all for test files, we could prefer a more specific file name, but this seems to work.
    "**/__support__/*",
    "**/*.stories.tsx"
  ]
}
```